src/main/java/com/vi/migrationtool/tenantservice/config/AgencyServiceJpaConfiguration.java

```diff
@@ -0,0 +1,49 @@
+package com.vi.migrationtool.tenantservice.config;
+
+import javax.sql.DataSource;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+@Configuration
+@EnableTransactionManagement
+public class AgencyServiceJpaConfiguration {
+
+  @Bean
+  @Primary
+  @ConfigurationProperties("spring.datasource")
+  public DataSourceProperties primaryDataSourceProperties() {
+    return new DataSourceProperties();
+  }
+
+  @Bean
+  @Primary
+  @ConfigurationProperties("spring.datasource")
+  public DataSource primaryDataSource() {
+    return primaryDataSourceProperties().initializeDataSourceBuilder().build();
+  }
+
+  @Bean
+  @Qualifier("agencyServiceDataSource")
+  @ConfigurationProperties("spring.datasource.agencyservice")
```

Comment:
I think this entry is missing in application.properties.
Please also remember to put appropriate changes in kubernetes repo (helm resources), to pass these configs to migrationService

Reply:
I believe we just need this in the Kubernetes repo, not here.